src/test/java/uk/gov/pay/products/stubs/publicapi/PublicApiStub.java

```diff
@@ -100,6 +101,7 @@ public static JsonObject createPaymentResponsePayload(String paymentId, long amo
                                                 .add("href", "https://an.example.link/from/payment/platform")
                                                 .add("method", "GET")))
                 .add("card_brand", "Visa")
+                .add(RandomStringUtils.randomAscii(8), "To test deserialisation handles new/unknown properties")
```

Comment:
This seems unnecessarily nondeterministic. Can't we just have a fixed string that we will never add to our API?

Reply:
I was worried someone might add the fixed string to `PaymentRequest`!

Reply:
But in general I’m in favour of deterministic tests, yes.